packages/core/src/components/dialog/_multistep-dialog.scss

```diff
@@ -110,13 +110,18 @@ $step-radius: $pt-border-radius * 2 !default;
     background-color: $pt-dark-icon-color-disabled;
   }
 
-  .#{$ns}-active & {
+  &.#{$ns}-active {
     background-color: $blue4;
   }
+
+  &.#{$ns}-dialog-step-viewed {
+    background-color: $gray3;
```

Comment:
Does this need to be different from the not-yet-viewed ones? If so: Might need dark support.

Reply:
Yes, the intent is that previously viewed steps are a darker gray because, unlike not-yet-viewed steps, you can click on them to navigate to that step. I think the gray3 looks fine in dark mode though:
![image](https://user-images.githubusercontent.com/7877370/114427655-4d061080-9b89-11eb-9714-68f5902a787e.png)